pkg/kubectl/custom_column_printer_test.go

```diff
@@ -206,6 +207,27 @@ func TestNewColumnPrinterFromTemplate(t *testing.T) {
 	}
 }
 
+func TestNewColumnPrinterFromSpecWithNoHeaders(t *testing.T) {
```

Comment:
Suggest adding no-headers tests in `TestNewColumnPrinterFromSpec`. You can add `noHeaders bool` as a field of the tests struct.